br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -40,13 +40,14 @@ const (
 		CREATE SCHEMA IF NOT EXISTS %s;
 	`
 
-	syntaxErrorTableName = "syntax_error_v1"
-	typeErrorTableName   = "type_error_v1"
+	syntaxErrorTableName = "syntax_error_v2"
+	typeErrorTableName   = "type_error_v2"
 	// ConflictErrorTableName is the table name for duplicate detection.
-	ConflictErrorTableName = "conflict_error_v1"
+	ConflictErrorTableName = "conflict_error_v1_2"
```

Comment:
Because the table structure diverse with master branch, it's not the same as `conflict_error_v4`. So I use `conflict_error_v1_2` here